VERSIONS.md

```diff
@@ -1,5 +1,6 @@
 | Version | iOS version | Android version | Common version | Unity IAP compatibility  |
 |---------|-------------|-----------------|----------------|--------------------------|
+| 5.0.0 | 4.25.0 | 6.7.0 | 5.4.0 | Unity IAP 4.4.0+ |
```

Comment:
Do we need to update the `Unity IAP` compatibility here? I think this should now be 4.8.0+ https://docs.unity3d.com/Packages/com.unity.purchasing@4.8/changelog/CHANGELOG.html, which is the first version where unity added support for BC5

Reply:
Good call!